securedrop/models.py

```diff
@@ -400,13 +428,31 @@ def set_password(self, passphrase):
 
         self.passphrase_hash = argon2.using(**ARGON2_PARAMS).hash(passphrase)
 
+    def set_name(self, first_name, last_name):
```

Comment:
We check the min length is greater than zero which is always true. If zero is an acceptable length, we sjould just remove this code path (and the same in the exceptions and validations and such).